tests/data/hosts

```diff
@@ -0,0 +1,25 @@
+#
+# hosts         This file describes a number of hostname-to-address
+#               mappings for the TCP/IP subsystem.  It is mostly
+#               used at boot time, when no name servers are running.
+#               On small systems, this file can be used instead of a
+#               "named" name server.
+# Syntax:
+#    
+# IP-Address  Full-Qualified-Hostname  Short-Hostname
+#
+
+127.0.0.1   localhost
+
+# special IPv6 addresses
+::1             localhost ipv6-localhost ipv6-loopback
+
+fe00::0         ipv6-localnet
+
+ff00::0         ipv6-mcastprefix
+ff02::1         ipv6-allnodes
+ff02::2         ipv6-allrouters
+ff02::3         ipv6-allhosts
+
+# other hosts
+192.168.178.1   some.box
```

Comment:
```suggestion
192.0.2.1   some.box
```
Let's use a bit more neutral address ([rfc5737](https://datatracker.ietf.org/doc/rfc5737/))

Reply:
ok, makes sense. Thanks I will change that

Reply:
done